openid-sharedsignals-framework-1_0.md

```diff
@@ -2036,6 +2128,15 @@ Receivers MUST tolerate receiving events for subjects that have been removed
 from the stream, and MUST NOT report these events as errors to the Event
 Transmitter.
 
+## Encrypting SETs {#management-sec-encrypting-sets}
+If an Event Transmitter is sending events that contain Personally Identifiable Information
+(PII), whether in the body of the event or in the `sub_id` of the SET itself, then the
+Transmitter MUST encrypt the SET such that it is sending a JWE instead of a JWT, as
```

Comment:
While I generally agree that PII should be encrypted, is this really something that the spec should require?

Reply:
The SET spec already requires it. See the second paragraph here: https://datatracker.ietf.org/doc/html/rfc8417#section-5.1